models/marts/ledger_current_state/contract_code_current.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+
+models:
+  - name: contract_code_current
+    description: '{{ doc("contract_code_current") }}'
+    
+    columns:
```

Comment:
nit: fill out the rest of the columns

Reply:
We can actually fill this out in a separate PR by trying dbt [codegen](https://hub.getdbt.com/dbt-labs/codegen/latest/)